packages/engine/Source/Scene/VoxelPrimitive.js

```diff
@@ -1059,6 +1059,15 @@ Object.defineProperties(VoxelPrimitive.prototype, {
       return this._customShaderCompilationEvent;
     },
   },
+
+  /**
+   * @private
```

Comment:
I know it's this way in `Cesium3DTileset`, but since it is helpful for internal developers to have type information and documentation too, lets try to document private properties with a similar rigor as public ones. 

Along the lines of:
```suggestion
   *  Loading and rendering information for requested content
   * @type {Cesium3DTilesetStatistics}
   * @readonly
   * @private
```